cypress-io/cypress
# fix: clean up inconsistencies in UI between sentence case and title case

- Closes https://github.com/cypress-io/cypress/issues/21854


### User facing changelog

Clean up inconsistencies in UI between sentence case and title case.

### Additional details
It has always been the intention of the Cypress Team to use `Sentence case` as opposed to `Title Case`.


### User facing changelog
- All `Title Case` have been converted to `Sentence case`.

### Steps to test
- Navigate the UI, 
- Verify that we are using `Sentence case` rather than `Title Case`.


### How has the user experience changed?


#### Before
https://www.loom.com/share/3d0d81f345254783b1f1d7c4ccf285b8

#### After
https://www.loom.com/share/ff1b3d98a04a4df2884c2c1c647df8a6

### PR Tasks
